src/search.ts

```diff
@@ -33,6 +34,12 @@ export default class Search extends SearchUtils implements SearchInterface {
      */
     constructor(userId: string, key: string) {
         super();
+        logger.info(`-------------------- Starting Swift-Search --------------------`);
+        const versionInfo = {
+            indexVersion: searchConfig.INDEX_VERSION,
+            supportedVersion: SEARCH_SUPPORTED_VERSION,
+        };
+        process.env.SWIFT_SEARCH = JSON.stringify(versionInfo);
```

Comment:
Is there any way that this data is available without a Symphony login? Because, if it is, a malicious user may use this info to track vulnerabilities in the underlying frameworks/libraries that we are using to plot attacks.